p2p/server_test.go

```diff
@@ -471,7 +470,7 @@ func TestServerSetupConn_whenNotInRaftCluster(t *testing.T) {
 			PrivateKey:  srvkey,
 			NoDiscovery: true,
 		},
-		newTransport: func(fd net.Conn) transport { return newTestTransport(clientpub, fd) },
+		newTransport: func(fd net.Conn, key *ecdsa.PublicKey) transport { return newTestTransport(clientpub, fd, key) },
```

Comment:
was this added by geth?

Reply:
by a big refactoring https://github.com/ethereum/go-ethereum/pull/21464